ci-operator/step-registry/openshift/e2e/aws/ovn/openshift-e2e-aws-ovn-workflow.yaml

```diff
@@ -11,5 +11,7 @@ workflow:
     - chain: gather-network
     - chain: gather-core-dump
     - chain: ipi-deprovision
+    env:
```

Comment:
Why are we only doing this for aws Jobs?

Reply:
baby steps. I don't want to get something merged that all of a sudden starts failing a ton of jobs. I have a ticket to
investigate why the core dumps are coming and will start working that. If we get this in and get progress on the
real cause of the core dumps we can enable this on more platforms. At least that was my thought process.